.github/workflows/ci.yml

```diff
@@ -424,6 +428,52 @@ jobs:
           name: airflow-provider-readmes
           path: "./files/airflow-readme-*"
 
+  helm-python-tests:
+    timeout-minutes: 5
+    name: "Python unit tests for helm chart"
+    runs-on: ubuntu-latest
+    needs: [build-info, ci-images]
+    env:
+      MOUNT_LOCAL_SOURCES: "true"
+      RUN_TESTS: true
+      TEST_TYPE: "Helm"
+      BACKEND: "None"
```

Comment:
You can also use SQLite. That shouldn't cause any problems in our case. This is also the default database used in Breeze.